packages/gestalt/src/Masonry.js

```diff
@@ -87,6 +87,7 @@ type Props<T> = {|
    * This is required if the grid is expected to be scrollable.
    */
   scrollContainer?: () => HTMLElement,
+  deferScrollPositionUpdate?: boolean,
```

Comment:
A couple of small requests:
- Can you add a `_` prefix to the prop name? We're in the process of converting our docs props tables to auto-generate from the component Props types, and we're likely to implement screening out undocumented props by that prefix.
- Can you add an explanatory comment with context around this prop?

Reply:
Friendly callout that you'll need to disable the dangling eslint rule when you do that: 
![Screen Shot 2021-11-16 at 4 21 08 PM](https://user-images.githubusercontent.com/7406105/142086932-ab43cc25-48fa-4e87-a668-f053066b4df5.png)